lib/Parse/Snort/Strict.pm

```diff
@@ -0,0 +1,42 @@
+package Parse::Snort::Strict;
+use base qw(Parse::Snort);
+
+use strict;
+use warnings;
+use Carp qw(croak);
+use List::Util qw(any);
+use Sub::Util qw(set_subname);
```

Comment:
I would also declare this dependency in the Makefile.PL, looking at https://metacpan.org/changes/distribution/Scalar-List-Utils you'll want to have a minimum version of 1.40.